modules/rosa-sts-interactive-cluster-creation-mode-options.adoc

```diff
@@ -51,7 +51,14 @@ The ROSA with Hosted Control Planes functionality is currently offered as a Tech
 |Specify if you want to use a pre-configured OIDC configuration or if you want to create a new OIDC configuration as part of the cluster creation process.
 
 |`Tags (optional)`
-|Specify a tag that is used on all resources created by ROSA in AWS. Tags are comma separated, for example: "key value, foo bar".
+|Specify a tag that is used on all resources created by {product-title} in AWS. Tags are comma separated, for example: "key value, foo bar".
```

Comment:
To imply that this is same as tags, suggest to add a statement that 'Tags can help you manage, identify, organize, search for, and filter resources within AWS'.

Reply:
> To imply that this is same as tags, suggest to add a statement that 'Tags can help you manage, identify, organize, search for, and filter resources within AWS'.

Thanks @bchandra-ocp. Have made that change.

Reply:
@bchandra-ocp Can you please give this a LGTM if you are happy with it? Thanks!